test/e2e/ytsaurus_controller_test.go

```diff
@@ -261,7 +260,156 @@ var _ = Describe("Basic test for Ytsaurus controller", func() {
 			"Should run and update Ytsaurus to the next major version",
 			getSimpleUpdateScenario("test-major-update", ytv1.CoreImageNextVer),
 		)
+		It(
+			"Should be updated according to UpdateSelector=Everything",
+			func(ctx context.Context) {
```

Comment:
Weird indentation. You can put It("description", func(...) {" into one line.

If linter complains about long line we can tweak it for tests.